### DISCUSSION TITLE
Swagger short code issue

### DISCUSSION THREAD

**Discussion by prdsoftware** [google/docsy#478] [General]
Hi all,

I'm trying to use the Swagger shortcode in my Docsy docs as per https://www.docsy.dev/docs/adding-content/shortcodes/#swaggerui, but can't get it to work.   In the browser console I'm getting the error:

> 
> Uncaught ReferenceError: SwaggerUIBundle is not defined
>     onload http://localhost:1313/docs/extending/web-api/swagger/:5414
>     EventHandlerNonNull* http://localhost:1313/docs/extending/web-api/swagger/:5413

Do I have to download some files and put them somewhere?. eg. the vendor dir?  ... or something else?

**Update:**

OK, I can see that there is a /themes/docsy/static/js dir that has swagger-ui-bundle.js and swagger-ui-standalong-preset.js

Seems like these are the files I'm after, but for some reason, Docsy is not looking here perhaps?

There is another "static" folder that is at the same level as the  config.toml file.  This is where my images/css is.  Is this confusing Docsy?

**Comment by narrenfrei** [comment]
Have you set the type of your page containing the Swagger shortcode to "swagger"?

**Reply by prdsoftware** [reply]
Oh...  that was it.   Easy.  Thank you for the prompt response that nailed it in one.   Well done.
